src/views/Home/components/Landing/index.tsx

```diff
@@ -29,7 +31,25 @@ const Landing: React.FC = () => {
 
         <div className='landing-component__section'>
           <div className='landing-component__text-box'>
-            <div className='landing-component__title'>CruzHacks 2022</div>
+            <div
+              className='landing-component__title'
+              role='button'
+              tabIndex={0}
+              onClick={() => {
+                setIsPuzzleHidden(false)
+              }}
+              onKeyDown={() => {
+                setIsPuzzleHidden(false)
+              }}
+            >
+              CruzHacks 2022
+            </div>
+            <PuzzleModal
+              question='Unscramble this word: actakhohn'
+              answer='hackathon'
+              display={isPuzzleHidden}
+              handleExit={() => setIsPuzzleHidden(true)}
+            />
```

Comment:
Could we update this Code so it passes in that div as children and still retains this property. Main reason I'm asking this is so once CruzHacks is over and we close some services, it will be easier to undo our changes.